javascript/script.js

```diff
@@ -8,18 +8,19 @@ let newItemInput;
 
 const $ = id => document.getElementById(id);
 const setToDoList = list => toDoList = list;
-
+let searchInput = $("search-text");
+const validate = () => {
+	//modal display on empty input
+	if (newItemInput.value.trim() === "") {
+		showEmptyInputModal();
+		return true;
+	}
+}
```

Comment:
If you return a value from anywhere in a function, make sure that function will return an apt value in every different scenario. Refer the [consistent return rule](https://eslint.org/docs/rules/consistent-return).